Server/VikingVault.Services/Abstractions/IBankAccountService.cs

```diff
@@ -11,5 +11,7 @@ public interface IBankAccountService
         List<BankAccount> GetBankAccounts(string token);
         void UpdateBankAccount();
         BankAccount ChangeBalance(UpdateBankAccountModel updatedBankAccount);
+        BankAccount AddMoneyToAccount(User user, string currency, float amountToBeAdded);
```

Comment:
How comes that you named a method "AddMoneyToAccount" but no account was sent as parameter?

Reply:
In this case, I am using the method in the controller, so I wanted to pass the responsibility of creating the account model to the method. I thought of adding only one BankAccount parameter to the method, but in the end I chose to do it this way beacause it reduces the number of lines in the top level code. In this case do you suggest changing the method name or the signature?